mjpc/tasks/humanoid/interact/contact_keyframe.cc

```diff
@@ -0,0 +1,55 @@
+#include "contact_keyframe.h"
+
+namespace mjpc::humanoid {
+    void ContactPair::Reset() {
+        body1 = NOT_SELECTED;
+        body2 = NOT_SELECTED;
+        geom1 = NOT_SELECTED;
+        geom2 = NOT_SELECTED;
+        for (int i = 0; i < 3; i++) {
+            local_pos1[i] = 0.;
+            local_pos2[i] = 0.;
+        }
+    }
+
+    void ContactKeyframe::Reset() {
+        name.clear();
+        for (auto& contact_pair : contact_pairs)
+            contact_pair.Reset();
+        
+        facing_target.clear();
+        weight.clear();
+    }
+
+    void to_json(json& j, const ContactPair& contact_pair) {
```

Comment:
Please use the Google C++ style guide convention for all [function names](https://google.github.io/styleguide/cppguide.html#Function_Names). For example, `to_json` -> `ToJson`

Reply:
These function names are used by [the JSON library ](https://github.com/nlohmann/json?tab=readme-ov-file#basic-usage) and need to adhere to that naming convention. But I can also use macros instead. For now I've removed this code, but in the future PRs I'll make sure to substitute this with macros.